ecommerce/CartController.php

```diff
@@ -166,6 +166,40 @@ public function get_cart_by_id() {
 		);
 	}
 
+	/**
+	 * Get cart items
+	 *
+	 * @since 3.0.0
+	 *
+	 * @return array
+	 */
+	public function get_cart_items() {
+		return $this->model->get_cart_items();
+	}
+
+	/**
+	 * Get delete course from cart
+	 *
+	 * @since 3.0.0
+	 *
+	 * @return void JSON response
+	 */
+	public function tutor_course_from_cart() {
+		if ( ! tutor_utils()->is_nonce_verified() ) {
+			$this->json_response( tutor_utils()->error_message( 'nonce' ), null, HttpHelper::STATUS_BAD_REQUEST );
```

Comment:
`$this->json_response( tutor_utils()->error_message( 'nonce' ), null, HttpHelper::STATUS_BAD_REQUEST );` I think this response is for API not for typical ajax request. Just use:
`wp_send_json_error( tutor_utils()->error_message( 'nonce' ) );`

Reply:
Updated.